portal/eproms_substudy_tailored_content/src/js/Base.js

```diff
@@ -0,0 +1,466 @@
+import NavMethods from "./Nav";
+import VideoMethods from "./Video";
+import {checkIE, getUrlParameter, tryParseJSON, PromiseAllSettledPolyfill} from "./Utility";
+
+export default {
+    created() {
+        PromiseAllSettledPolyfill();
+    },
+    mounted() {
+        Promise.allSettled([
+            this.$http(this.settingsURL).catch(error => { return error }),
+            this.$http(this.meURL).catch(error => { return error }),
+            this.$http("//geoip.cirg.washington.edu/json/"),
```

Comment:
Would you be able to make the URL for the GeoIP service configurable, defaulting to the current URL? If possible, it's better to omit hardcoded references to specific infrastructure in code, so it's easier to change in one place.

Reply:
thank you, fixed